src/SmartComponents/SystemInventory/SystemInventoryCard.js

```diff
@@ -39,47 +45,62 @@ const SystemInventoryCard = ({
+                            <NumberDescription
+                                data={ inventorySummary.total.toLocaleString() || 0 }
+                                dataSize="lg"
+                                percentageData={ intl.formatMessage(messages.systemInventoryPercentageData,
+                                    { count: Math.floor((inventorySummary.total / inventoryTotalSummary.total) * 100) || 0 }
+                                ) }
+                                linkDescription = { intl.formatMessage(messages.systemInventoryDescription,
+                                    { count: inventorySummary.total || 0 }
+                                ) }
+                                link='./insights/inventory'
+                                iconTooltipText = { intl.formatMessage(messages.systemInventoryNotUsingClient,
+                                    { count: inventoryTotalSummary.total - inventorySummary.total || 0 }
+                                ) }
+                            />
+                        }
+                        { inventoryStaleFetchStatus === 'fulfilled' &&
+                            <Button component="a" variant="link" href='./insights/inventory/?status=stale' isInline>
+                                <IconInline
+                                    message={ intl.formatMessage(messages.systemInventoryStale,
+                                        { count: inventoryStaleSummary.total || 0 }
+                                    ) }
+                                    state="warning"
+                                    systemInventory="true"
+                                />
+                            </Button>
+                        }
+                        { inventoryWarningFetchStatus === 'fulfilled' &&
+                            <Button component="a" variant="link" href='./insights/inventory/?status=stale_warning' isInline>
+                                <IconInline
+                                    message={ intl.formatMessage(messages.systemInventoryStaleWarning,
+                                        { count: inventoryWarningSummary.total || 0 }
+                                    ) }
+                                    state="critical"
+                                    systemInventory="true"
+                                />
+                            </Button>
+                        }
+                        { inventoryTotalFetchStatus === 'rejected' &&
+                            <FailState appName='Inventory' isSmall/>
+                        }
+                    </Fragment> : <NotAuthorized
+                        showReturnButton={ false }
+                        serviceName="Inventory"
+                        description={ <Fragment>
+                            <div>Your organization administrator must grant</div>
+                            <div>you inventory access to view your systems.</div>
```

Comment:
What about to wrap them via using CSS?

Reply:
We want to break the sentence on specific place, at least that's on mocks.

Reply:
Yeah, I understand, I know that very well. :smiley:  But you should be able to setup the same width/max-width as in the mocks and the wrapping should correspond too.

Reply:
But it's definitely no blocker or whatever. (It can cause some troubles if we are going to translate this message _in the future_.)

Reply:
I gotta ditto the translation thing, please throw the strings in Messages.js n use the `intl.formatMessage(` thing

Reply:
Good poin! I've added the string in there and also styled it so it would fit the mocks and not used 2 divs.